schemas/resources/schemas/collection-draft/v1.0.0/metadata.json

```diff
@@ -436,7 +436,7 @@
         "Date" : "2020-03-02T00:00:00.000Z",
         "Type" : "UPDATE"
       }, {
-        "Date" : "2038-01-02T00:00:00.000Z",
+        "Date" : "2037-12-31T00:00:00.000Z",
```

Comment:
this is to avoid the https://en.wikipedia.org/wiki/Year_2038_problem

Reply:
TIL about this problem